docs/errors.rst

```diff
@@ -148,82 +148,89 @@ Group 1 - Api Request
+| PGRST110      |             | See :ref:`limited_update_delete`.                           |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst111: | 500         | An invalid ``response.headers`` was set.                    |
+|               |             | See :ref:`guc_resp_hdrs`.                                   |
+| PGRST111      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst112: | 500         | The status code must be a positive integer.                 |
+|               |             | See :ref:`guc_resp_status`.                                 |
+| PGRST112      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst113: | 406         | More than one column was returned for a scalar result.      |
+|               |             | See :ref:`scalar_return_formats`.                           |
+| PGRST113      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst114: | 400         | For an :ref:`UPSERT using PUT <upsert_put>`, when           |
+|               |             | :ref:`limits and offsets <limits>` are used.                |
+| PGRST114      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst115: | 400         | For an :ref:`UPSERT using PUT <upsert_put>`, when the       |
+|               |             | primary key in the query string and the body are different. |
+| PGRST115      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst116: | 406         | More than 1 or no items where returned when requesting      |
+|               |             | a singular response. See :ref:`singular_plural`.            |
+| PGRST116      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst117: | 405         | The HTTP verb used in the request in not supported.         |
+|               |             |                                                             |
+| PGRST117      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst118: | 400         | Could not order the result using the related table because  |
```

Comment:
I have a doubt about this error. It is also used when a column is not found when using `column`. These are [the lines of code](https://github.com/PostgREST/postgrest/blob/eb3f7c696a5da8c023ec090eb16fa05c87adf339/src/PostgREST/Error.hs#L214-L218).

Reply:
@laurenceisla Yes, that needs to be corrected. It slipped by when reviewing https://github.com/PostgREST/postgrest/pull/2542

Reply:
Fixed in https://github.com/PostgREST/postgrest/pull/2648